### DISCUSSION TITLE
Remember the repository we just used

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#116482] [Mobile] [answered]
We have just created an issue through sharing an image into the GitHub mobile app. 

Now we are back again sharing another image into  the mobile app. 

The problem is on this list of choices we don't see the choice for the repository that we used just moments ago for the previous issue. 

We have to search for it all over again by entering its name.

If I were in charge here, it would be right there on the top of the list!

![Screenshot_20240402_092452_GitHub.jpg](https://github.com/community/community/assets/1230959/01a92031-ae18-4ca3-9759-c2375f056ebb)

**Comment by eliperkins** [comment, marked as answer]
Hey @jidanni , this is great feedback! I can see how it'd be helpful to have the repository that you most recently created an issue in be at the top of the list when creating new issues.

I've shared this feedback with the team, and I'll be sure to keep you posted on progress towards this improvement. 🙇
